lib/resources/example_json/r4_examples_care_plan.rb

```diff
@@ -124,5 +186,135 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_CARE_PLAN_PATIENT_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '013ed260-cdd4-4ffc-947e-aeadb699d3d4',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/CarePlan?patient=12724066&category=assess-plan'
+        }
+      ],
+      'entry': [
+        {
```

Comment:
Can we use R4_CARE_PLAN_PATIENT_ENTRY here

Reply:
Fixed in the commit 8f2aa57